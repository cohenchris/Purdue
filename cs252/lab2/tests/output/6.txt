0x56394ce65060
(nil)
[
	addr: 0096
	size: 3968
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x56394ce650b0
(nil)
[
	addr: 0176
	size: 3888
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x56394ce65100
(nil)
[
	addr: 0256
	size: 3808
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x56394ce65150
(nil)
[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
0x56394ce65010
0x56394ce65150
[
	addr: 0016
	size: 64
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0336
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: 0016
	next: NULLPTR
]

0x56394ce65010
0x56394ce65150
[
	addr: 0016
	size: 144
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0336
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: 0016
	next: NULLPTR
]

0x56394ce65010
0x56394ce65150
[
	addr: 0016
	size: 224
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0336
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: 0016
	next: NULLPTR
]

BREAK
0x56394ce65010
(nil)
[
	addr: 0016
	size: 4048
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
