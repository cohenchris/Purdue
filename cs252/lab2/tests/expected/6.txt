
(nil)
[
	addr: 0096
	size: 3968
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0176
	size: 3888
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0256
	size: 3808
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK


[
	addr: 0016
	size: 64
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0336
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: 0016
	next: NULLPTR
]



[
	addr: 0016
	size: 144
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0336
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: 0016
	next: NULLPTR
]



[
	addr: 0016
	size: 224
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0336
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: 0016
	next: NULLPTR
]

BREAK

(nil)
[
	addr: 0016
	size: 4048
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
